core/src/test/java/org/apache/druid/data/input/impl/InputRowParserSerdeTest.java

```diff
@@ -52,7 +52,8 @@ public void testStringInputRowParserSerde() throws Exception
             new TimestampSpec("timestamp", "iso", null),
             new DimensionsSpec(DimensionsSpec.getDefaultSchemas(ImmutableList.of("foo", "bar")), null, null),
             null,
-            null
+            null,
+            false
```

Comment:
nit: it would be easier to read if you made false a constant at the top of this test, like `DO_NOT_KEEP_NULL_COLUMNS`